rest/src/main/java/com/redhat/ipaas/rest/v1/handler/BaseHandler.java

```diff
@@ -25,6 +25,13 @@
     @Autowired
     private DataManager dataMgr;
 
+    public BaseHandler() {
+    }
+
+    public BaseHandler(DataManager dataMgr) {
```

Comment:
Wouldn't it be better to use constructor injection here (as we do in many other places, too ?). This would require to change subclasses to call the parent constructor is probably more straight forward.

If you don't want to bother on this level of details, I'm happy with it for now ;-)